padertorch/contrib/examples/speech_enhancement/simple_mask_estimator/README.md

```diff
@@ -0,0 +1,48 @@
+Simple Mask Estimator
+=============
+
+This directory contains scripts to train and evaluate a simple mask estimator
+inspired by .
+
+Results
+-------
+
+The simple mask estimator model can achieve the following results:
```

Comment:
Can you mention, that it is on the Chime3 6ch track?
Are the lines `observed` and `masked` on the 1ch track or on the first channel of the 6ch track?

Reply:
yes, I'll add that, its on the first channel of the 6ch track

Reply:
done